kobo/apps/subsequences/tests/test_submission_stream.py

```diff
@@ -257,7 +258,8 @@ def test_stream_with_extras_handles_duplicated_submission_uuids(self):
                 '_uuid': '1c05898e-b43c-491d-814c-79595eb84e81',
             },
         ]
-        self.asset.deployment.mock_submissions(submissions)
+        with self.assertRaises(ConflictingSubmissionUUIDError):
+            self.asset.deployment.mock_submissions(submissions)
```

Comment:
if `self.asset.deployment.get_submissions` returns only submission, the test is useless.
We may need to find another way to make the test pass. We may need to mock the data only for this test to bypass the unique constraint.